### ISSUE TITLE
Improve "Portfolio" icon to aid communication between traders

### ISSUE THREAD

**Issue by Conza88** [bisq-network/bisq#5074]
Hi team,

Goal is to improve communication between trading parties, resulting in swifter trades, and less issues/mediation requests etc. as a result of an improved notification that there is an unread trader message pending a response.

This red icon displays the number of open trades. For me it remains red regardless if actioned or responded to or not. 

I think it would be helpful to utilise this better - signifying when a response or action has been received? I know there are pop-ups for various other results to prompt action which is great, but not for chat as far as I am aware. 

![image](https://user-images.githubusercontent.com/51952482/104179064-02c0a680-5457-11eb-9e24-1472036e3d47.png)

Perhaps it goes _green_ when all read/actioned? And _red_ - when there is an unread chat message? 

Or maybe "orange" [BTC] as normal/standard (i.e. open trade pending), and _red_ when an unread message has been received? 

Open to ideas on whether to help notify traders there's a new message to respond to. Often for myself, I check regularly but would have no idea otherwise.

**Comment by ripcurlx** [comment]
Yes, that is on my TODO list for a long time to have a "real" notification badge that only indicates if you have actually something to do and which decreases the counter after the action was done.

**Comment by pazza83** [comment]
Also any way for users to close mediation cases?

I have a couple open now that have been settled through mediation but the cases are still open. Users can re-open cases so I think allowing them to close cases should not prevent too much issues.

There is also a bug in the app that does not sort by status. So closed / open mediation tickets are not sorted correctly. Makes it difficult to manage if you have more than a screenful with open tickets that are actually closed mixed in.

**Comment by Conza88** [comment]
Yeah, great points
